### PR TITLE
Parameterize $NPM for cut.sh

### PR DESCRIPTION
**Pull request by kriskowal**

### CODE REVIEW

**Comment by kriskowal** [comment]
r @jcorbin

File: node/cut.sh [hunk 1]
```diff
@@ -66,6 +66,11 @@ git tag -a -m "Tag $tag" "$tag" "$DEV_BRANCH"
...
 git push -f origin dev_node
 
+# kris (at least) does not use the default npm for the public registry
+if [ "$NPM" == "" ]; then
+    NPM=npm
+fi
+
 git archive --prefix=package/ --format tgz dev_node >package.tgz
-npm publish package.tgz --tag "${NPM_TAG:-alpha}"
+$NPM publish package.tgz --tag "${NPM_TAG:-alpha}"
```
Comment (jcorbin) [inline node/cut.sh]: Just please use defaulting syntax like we do for $NPM_TAG, so `${NPM:-npm}`.  Also please update hyperbahn/cut.sh while you're at it.

**Comment by kriskowal** [comment]
revised

**Comment by jcorbin** [comment]
:100:

**Pull request merged**